IANotificationsTestProject/In App Notifications/IANotification.swift

```diff
@@ -0,0 +1,89 @@
+import UIKit
+
+//IANotification is where the content of an in app notification is stored
+//This class contains the window that displays the notification and all of the UI elements
+//This class handles triggering actions if present and tapped
+
+class IANotification: UIViewController {
```

Comment:
Note about the naming!!

`IANotification` sounds like a Model view (not a ViewController?)

Reply:
To me it reads like interface file from java :-)